src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -60,7 +60,7 @@ This resource contains explanations for the various functions and clauses availa
 
 * [Intro to NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql): provides a brief overview of NRQL as a whole, and links to other introductory docs
 * [How to use NRQL](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql-how-nrql-works): explains how NRQL works and how to use it to query data
-* [Introductory NRQL tutorial](/docs/query-data/nrql-new-relic-query-language/getting-started/introduction-nrql-tutorial): a basic tutorial for NRQL designed to get you familiar with basic queries
+* [Introductory NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/): a basic tutorial for NRQL designed to get you familiar with basic queries
```

Comment:
Does this need the trailing `/` for any reason?

Reply:
No, it doesn't. That's just how the path copied from where I grabbed it my browser. I think it will work either way at this point...

Reply:
No, I just added it out of habit :)